core/mutation/mutation.go

```diff
@@ -28,6 +28,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
+	spb "github.com/google/keytransparency/impl/proto/mutation_v1_service"
```

Comment:
Sure, as discussed on hagouts, I was already aware of this. I've created this PR early because I wasn't sure why not even the non-stream version `GetMutations` is exposed by these changes. It should already, or not?